WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell+ViewModel.swift

```diff
@@ -62,6 +63,11 @@ extension AddressTableViewCell.ViewModel {
             value: "Sale",
             comment: "The 'Sale' label under the domain name in 'Choose a domain' screen"
         )
+        static let freeWithPaidPlan = NSLocalizedString(
+            "domain.suggestions.row.free-with-plan",
+            value: "Free for a year with a plan",
```

Comment:
Did you have a chance to check this, @staskus?

Reply:
@guarani I have missed this comment. I used a shorter message so it would fit into the current layout. If we want to use the full length of the message we need to restructure this new UI, as is done on Android: https://github.com/wordpress-mobile/WordPress-Android/pull/19283#issuecomment-1740816603

Reply:
I will merge this PR so we won't have too many ongoing PRs but I think it's worth restructuring UI as is done on Android.

Reply:
Makes sense, thanks @staskus